### ISSUE TITLE
New notebook on unpin instead of pane

### ISSUE THREAD

**Issue by nhold** [nhold/wxWidgets#19]
There should be a style where the user drags or unpins a tab or pane out of the main frame and it creates another notebook and places itself inside of it similar to Unity3D or Visual Studio 2012 and up.

**Comment by ghost** [comment]
This should be easy to achieve now, as it is just what I now do inside wxAuiNotebook. I'll look at it once we will be merged back in wxWidgets.

**Comment by ghost** [comment]
My bad, I should have had a look to unity3d before. I thought to have a pane always dock as a tab; but you may want to be able to create notebooks in floating frames. This suppose to create additionnal wxAuiManager in miniframes on the fly. This is a bit more complicated, but not impossible.

**Comment by nhold** [comment]
Yeah, I haven't been able to figure out a nice way of doing it. Hopefully with both of us giving it a shot we can figure it out.

**Comment by ghost** [comment]
I was implementing a pane option to force a pane to always be in a notebook, and I discovered that the floating frame has itself a wxAuiManager inside; this implies that it should be possible to easily dock several panes in a floating frame. I propose however to postpone this after my next pull request and having reintegrated the main repository.

**Comment by nhold** [comment]
Agreed
